src/shared/proto/boxlite/v1/service.proto

```
@@ -383,13 +383,21 @@ message ContainerConfig {
+// exception — see the field's own comment below for why it stays flat.
 message ContainerAdvancedOptions {
-  ProcessOptions process = 1;
+  // Flat, not nested under a ProcessOptions submessage the way linux/mount
+  // are nested below: this field predates this message (added in #1047,
+  // guest version floor 0.9.8, long since deployed) as a direct
+  // ContainerCapabilities on field 1, not wrapped. Wrapping it now would
+  // reuse field 1's tag for a different wire shape — old, already-running
+  // guests would decode the new bytes against their old field-1 layout and
+  // get garbage. linux/mount are new in this feature with no such
+  // constraint, so they get the cleaner shape capabilities can't.
+  ContainerCapabilities capabilities = 1;
```

Buf (1.72.0)

[error] Field "1" with name "capabilities" on message "ContainerAdvancedOptions" changed type from "boxlite.v1.ProcessOptions" to "boxlite.v1.ContainerCapabilities".

(FIELD_WIRE_JSON_COMPATIBLE_TYPE)